méli-mélo/2024-04-stepsquiz/meta.md

```diff
@@ -0,0 +1,45 @@
+---
+feature: méli-mélo
+"@context":
+  "@version": 1.0
+  dct: http://purl.org/dc/terms/
+  title:
+    "@id": dct:title
+    "@container": "@language"
+  description:
+    "@id": dct:description
+    "@container": "@language"
+  modified: dct:modified
+
+title:
+  en: Quiz pattern
+description:
+  en: Apply a quiz pattern to the existing Steps Form component.
```

Comment:
Can you provide the French translation of the title and the description only. Thanks